holoviews/ipython/display_hooks.py

```diff
@@ -127,40 +121,64 @@ def option_state(element):
         if raised_exception:
             dynamic_optstate(element, state=optstate)
 
+
+def mimebundle_to_html(bundle):
+    """
+    Converts a MIME bundle into HTML.
+    """
```

Comment:
Minor and won't hold up the PR: would be good to say this is for adding to the notebook archive.